Source/Atomic/Scene/Node.h

```diff
@@ -495,7 +495,7 @@ class ATOMIC_API Node : public Animatable
     void RemoveChild(Vector<SharedPtr<Node> >::Iterator i);
     /// Return child nodes recursively.
     void GetChildrenRecursive(PODVector<Node*>& dest) const;
-    /// Return child nodes with a specific component recursively.
+	/// Return child nodes with a specific component recursively.
     void GetChildrenWithComponentRecursive(PODVector<Node*>& dest, StringHash type) const;

```

Comment:
Must have been prior to our switching to spaces.Do I need to correct my side and re-create the PR?